contracts/contracts/ProviderRegistry.sol

```diff
@@ -194,53 +189,37 @@ contract ProviderRegistry is
         require(successFee, "fee recipient transfer failed");
     }
 
-    /**
-     * @dev Withdraw funds to the bidder.
-     * @param bidder The address of the bidder.
-     */
-    function withdrawBidderAmount(address bidder) external nonReentrant {
-        require(bidderAmount[bidder] > 0, "Bidder Amount is zero");
-
-        bidderAmount[bidder] = 0;
-
-        (bool success, ) = bidder.call{value: bidderAmount[bidder]}("");
-        require(success, "Couldn't transfer to bidder");
+    /// @dev Requests withdrawal of the staked amount.
+    function requestWithdrawal() external {
```

Comment:
To be consistent with [here](https://github.com/primev/mev-commit/blob/d5df1718f53699ed7d50aa6a472e0c9b40277f81/contracts/contracts/validator-registry/ValidatorRegistryV1.sol#L137), this action could be called "unstake". Doesn't matter too much tho, that's just the language I'm used to with other chains